spec/zoneddatetime.html

```diff
@@ -1367,9 +1367,10 @@ <h1>
             1. Set _done_ to *true*.
         1. If _days_ &lt; 0 and _sign_ = 1, throw a *RangeError* exception.
         1. If _days_ &gt; 0 and _sign_ = -1, throw a *RangeError* exception.
-        1. If _nanoseconds_ &lt; 0 and _sign_ = 1, throw a *RangeError* exception.
+        1. If _nanoseconds_ &lt; 0, then
+          1. Assert: _sign_ is -1.
         1. If _nanoseconds_ &gt; 0 and _sign_ = -1, throw a *RangeError* exception.
-        1. If abs(_nanoseconds_) &ge; abs(_dayLengthNs_), throw a *RangeError* exception.
+        1. Assert: abs(_nanoseconds_) &lt; abs(_dayLengthNs_) is *true*.
```

Comment:
I think the linter's complaint here might be incorrect, but maybe there's a more ecmarkup-friendly way to phrase this?

Reply:
How about `Assert: The inequality abs(_nanoseconds_) &lt; abs(_dayLengthNs_) holds.`?

Reply:
Sgtm, will reword quickly.